definition_of_ready.md

```diff
@@ -0,0 +1,34 @@
+# Definition of ready (DoR)
+- Each user story should fulfill the INVEST criteria (see e.g. [Wikipedia](https://en.wikipedia.org/wiki/INVEST_(mnemonic)), [scrum-in-der-praxis.de](https://scrum-in-der-praxis.de/glossary/invest))
+  - See https://www.knowledgehut.com/tutorials/scrum-tutorial/definition-of-ready
+    - Independent (of all others)
+    - Negotiable (not a contract for a specific feature)
+    - Valuable (a vertical slice)
+    - Estimable (to a decent approximation)
+    - Small (enough to fit in a single iteration)
+    - Testable (even if the test doesn't exist yet)
+
+## Further requirements
+- The user story is Well-defined:
+  - The user story should be written in the user story format, see for example [Wikipedia](https://en.wikipedia.org/wiki/User_story#Principle).
+  - e.g. `As a <role> I can <capability>, so that <receive benefit>`
+- Acceptance criteria are defined (see point 7 from [dzone.com](https://dzone.com/articles/10-tips-for-writing-good-user-stories))
+  - The conditions that have to be fulfilled such that the story is done are described. The description should allow any member of the dev team to decide whether the story is done.
+  - The team knows how to "demonstrate" the user story to the user. In other words, the team knows the value for the user/stakeholder.
+  - "Without such criteria developers may not really understand the scope of the work or how it will be tested and validated." (Source: [scrum.org](https://www.scrum.org/resources/blog/walking-through-definition-ready))
+- Diagrams are used to support the understanding (see point 10 from [dzone.com](https://dzone.com/articles/10-tips-for-writing-good-user-stories))
+- No implicit knowledge
+  - All information regarding a User-story should be written down in easy to understand sentences. Reason: It only takes a few minutes to write valuable information down but it takes ages to fix implementations based on incorrectly understood/missing information.
+   - Whenever new knowledge is gained regarding the User-story, it should be written down in the User-story, to promote information exchange.
```

Comment:
```suggestion
- Easy to understand: The team writes down or attaches additional information, that it requires to understand a user-story.
```

Reply:
I would not write it like that, because I think ideally the final description of the user-story is worked out together.